File: Jr/2020/DogTreats.java
/**
 * Owen Bélanger-Love
 * 23 sept 2024
 */


// Import Scanner class
import java.util.Scanner;

public class DogTreats {
    public static void main(String[] args) {

        // Create input scanner
        Scanner input = new Scanner(System.in);
        
        // Get the amount of small treats
        int small = input.nextInt();
        

        // Get the amount of medium treats
        int medium = input.nextInt();

        // Get the amount of large treats
        int large = input.nextInt();

        // Check happiness rating
        if (small + 2 * medium + 3 * large >= 10) {

            // There is enough points
            System.out.println("happy");
        }

        else {

            // There is not enough points
            System.out.println("sad");
        }

        // Close input Scanner
        input.close();
    }
}

File: Jr/2020/DogTreats.py
# Owen Bélanger-Love

# Get the amount of small dog treats
small = int(input())

# Get the amount of medium dog treats
medium = int(input())

# Get the amount of large dog treats
large = int(input())

# Calculate happiness
happiness = small + 2 * medium + 3 * large

# Check if happiness score is high enough
if happiness >= 10:
    # Happy enough
    print("happy")

else:
    # Not happy enough
    print("sad")